tests/networkpolicy_test.go

```diff
@@ -215,3 +225,18 @@ func skipNetworkPolicyRunningOnKindInfra() {
 		Skip("Skip Network Policy tests till issue https://github.com/kubevirt/kubevirt/issues/4081 is fixed")
 	}
 }
+
+func waitForNetworkPolicyDeletion(policy *v1network.NetworkPolicy) {
+	if policy == nil {
+		return
+	}
+
+	virtClient, err := kubecli.GetKubevirtClient()
+	Expect(err).ToNot(HaveOccurred())
```

Comment:
nit: You forgot an offset here. (if you change something else, you can change this one as well, otherwise do not bother)